eslint.config.mjs

```
@@ -0,0 +1,165 @@
+import typescriptEslint from "@typescript-eslint/eslint-plugin";
+import tsdoc from "eslint-plugin-tsdoc";
+import _import from "eslint-plugin-import";
+import { fixupPluginRules } from "@eslint/compat";
+import globals from "globals";
+import tsParser from "@typescript-eslint/parser";
+import * as parser from "@graphql-eslint/eslint-plugin";
```

ESLint

[error] 'parser' is defined but never used.

(@typescript-eslint/no-unused-vars)